app/controllers/static_pages_controller.rb

```diff
@@ -1,4 +1,21 @@
 class StaticPagesController < ApplicationController
+
   def home
+    @card = Card.review_date_over(Time.now).random_card
```

Comment:
Зачем передавать текущее время, просто оставляй в скоупе и все

Reply:
Вначале так и хотел сделать, но потом перечитал: 
class Post < ActiveRecord::Base
  scope :published_last_week, where('published_at >= ?', 1.week.ago)
end
Because this won’t work as expected: 1.week.ago will be evaluated when the class is loaded, not every time the scope is called.
http://blog.plataformatec.com.br/2013/02/active-record-scopes-vs-class-methods/
из материалов к заданию и все переделал

Reply:
А какая разница?) Нам же только дата нужна. Убирай этот аргумент, это лишнее

Reply:
Ok. Done.
# card.rb
scope :review_date_over, ->{ where("review_date < ?", Time.now) }
# static_pages_controller.rb
@card = Card.review_date_over.random_card